_posts/2020-07-21-why-paper-and-pencil-is-my-best-tool.markdown

```diff
@@ -0,0 +1,48 @@
+---
+layout: post
+title:  "Why paper and pencil are my best tools"
+date:   2020-07-21 23:30:41 -0300
+categories: tools
+---
+Just try thinking about how much time we spend looking for an ideal tool to taking our notes, reminders, scheduling, sketching… A tool that give us freedom and flexibility to do everything we need. Yeah... so try looking around. Can you see this tool? There it is! A simple paper and pencil right next to you.
+
+I can’t imagine how many ideas and how much time I saved using this and I will try explain why this works so well to me.
+
+In all my programmer life I’ve tried to use many tools that could improve my capabilities to remember things, organize to do lists, improve creativity, etc. One thing that worked so well on all my needs giving me freedom, and combining all other tools that have separated and specialized functions, is paper and pencil.
+
+I can’t imagine how many ideas and how much time I've saved using just this tool, and I will try explain why this works so well to me.
+
+> Ok, But what about smartphone's on-screen notifications for your scheduled to do lists/meetings?
+
+Ok. I know and I use this too as an extra (Yeah... as an extra, not as the main tool)
+
+Organizing ideas on a paper or a notebook turned me more disciplined than ever, just because I can create mindsets, draw whatever I need to explore and remember myself about some topics with emphasis. This kind of things enforces our memory so well that there will never be a technology to do it in the same way
+
+![image](/assets/images/creativity-paper-1280.jpg)
+
+Although technology has its credits about many other things, and we are programmers because of it, I couldn't abandon paper and its freedom.
+
+I made many class diagrams, flowcharts and database modeling which saved me much time, instead of start coding right away, without thinking carefully of what I could really do.
+
+This past experience demonstrates that this action saves me much more time than if I had coded first and just then thought on how to solve my problem or applying any design patterns. It's OK to code directly if it's something easy, trying to draft this on paper could cost more time than code directly and that’s fine.
+
+Try it, and tell me later about your experiences.
+
+### Bullet Journal
+
+Another big change I've made on my life's organization methods is the use of Bullet Journal and I recommend you to know a bit about it.
```

Comment:
"I recommend you to know a bit about it."
should be:
"I recommend knowing a bit about it."